src/start.ts

```diff
@@ -47,11 +54,20 @@ export const handler = async (_event: any = {}): Promise<any> => {
 
   const providerPromises = flatMap(
     evmChains.map((chain: node.ChainConfig & ChainConfig) => {
-      return map(chain.providers, async (chainProvider, _) => {
+      return map(chain.providers, async (chainProvider, providerName) => {
```

Comment:
This is also a minor issue:
You can actually inline this, ie. remove the 
```
=> {
return 
```
So it becomes 
```
  const providerPromises = flatMap(
    evmChains.map((chain: ChainConfig & ChainConfig) =>
       map(chain.providers, async (chainProvider, providerName) => {
        const providerLogOptions = {
```

Reply:
I guess here I just wanted to have it look like in [Airnode](https://github.com/api3dao/airnode/blob/869fc03048e344a2efdc2fe5303f31c63db478e1/packages/airnode-node/src/providers/actions.ts#L54)